fe/fe-core/src/main/java/com/starrocks/catalog/Catalog.java

```diff
@@ -5863,27 +5886,41 @@ public void replayModifyTableProperty(short opCode, ModifyTablePropertyOperation
         db.writeLock();
         try {
             OlapTable olapTable = (OlapTable) db.getTable(tableId);
-            TableProperty tableProperty = olapTable.getTableProperty();
-            if (tableProperty == null) {
-                olapTable.setTableProperty(new TableProperty(properties).buildProperty(opCode));
+            if (opCode == OperationType.OP_SET_FORBIT_GLOBAL_DICT) {
+                String enAble = properties.get("enable_low_card_dict");
+                if (enAble == null) {
```

Comment:
Could use `Preconditions.checkState` here. normally, we won't throw `Error` directly.